src/internal/coredb/projects.go

```diff
@@ -14,35 +14,81 @@ import (
 	"github.com/pachyderm/pachyderm/v2/src/pfs"
 )
 
+const (
+	pageSize = 100
```

Comment:
Why make this a constant? We know pageSize can change.

Reply:
I made it a constant since I'm not sure if we are going to expose this to the user, but I suppose it doesn't have to be. I think for ListProjectTx we'll end up just exposing the pages